### DISCUSSION TITLE
Is it possible to display different entities depending on the request?

### DISCUSSION THREAD

**Discussion by Julien-degardin** [awcodes/overlook#11] [Q&A] [answered]
Hello guys!
Firstly, thank you for this package, it looks great and is easy to install 👍 

I have two pages that I use as "dashboards" where I'd like to show different entities depending on the page (or URL). 
Is it possible? 

Thank you!

PS: I tried to use 
```php 
if ($_SERVER['REQUEST_URI'] === "/admin/dashboard-stock") {
```
but I'm getting this error "array_merge(): Argument 2 must be of type array, int given"

**Comment by awcodes** [comment, marked as answer]
Where are you using the conditional?

That's tricky since widgets are Livewire components and they can't be instantiated. Meaning they can't be dynamically registered with options. You could probably create your own Widget that extends the Overlook widget and manually set the resources there. Then you could register the right widget in the context you need it it.

**Reply by Julien-degardin** [reply]
Thank you for the fast answer :)

I'm using the conditional in the config/overlook.php but I know this can't be the way to make it work.

Do you have any documentation which I could use to build my custom widget?

**Reply by awcodes** [reply]
No documentation. It's just standard OOP technique. Maybe google 'extending classes in php'.

**Reply by Julien-degardin** [reply]
Okay thanks, I'll give it a try! 

I'm marking this as closed.
